testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -528,6 +507,51 @@ QUnit.module('datebox tests', moduleConfig, () => {
     });
 });
 
+QUnit.module('toolbar buttons testing', {}, () => {
```

Comment:
If this module is names as a "testing", should we name all our tests modules as a "testing"?
```suggestion
QUnit.module('toolbar buttons', {}, () => {
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/22991/commits/6db8ca42fc7d248ad8eb6711a8299e46aae5347b